templates/index.html

```diff
@@ -11,7 +11,11 @@ <h2>Items</h2>
     <ul class="list-group mb-4">
       <!-- List the todo items here -->
       {% for item in items %}
-      <li>{{ item["id"] }} | {{ item["title"] }}</li>
+        <li>{{ item.name }} | {{ item.status }}</li>
+        <form method="POST" action="/complete-item">
+          <input type="text" name="id" value="{{ item.id }}" hidden/>
+          <button type="submit" type="button" class="btn btn-primary">Complete</button>
+        </form>
```

Comment:
You could wrap this form in an `{% if ... %} ... {% endif %}` that checks it's not already complete